TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Security/Authorization/Privilege/Node/CreateNodePrivilegeContext.php

```diff
@@ -24,7 +24,7 @@ class CreateNodePrivilegeContext extends NodePrivilegeContext
     protected $creationNodeTypes;
 
     /**
-     * @param string|array $creationNodeTypes either an array of supported node type identifiers or a single node type identifier (for example "TYPO3.Neos:Document")
+     * @param string|array $creationNodeTypes either an array of supported node type identifiers or a single node type identifier (for example "Neos.Neos:Document")
```

Comment:
This is not correct yet and should only be changed when the naming is actually changed in master only.